roles/test_observabilityStrategy/tasks/main.yml

```diff
@@ -0,0 +1,86 @@
+---
+# Assuming we've already logged in...
+# Following procedure on https://infrawatch.github.io/documentation/#configuring-observability-strategy_assembly-installing-the-core-components-of-stf
+# Assuming we're in teh right project already...
+
+
+- name: "Add observabilityStrategy: none to the spec section"
+  ansible.builtin.shell:
+    cmd: |
+      oc patch stf/default --type merge -p '{"spec":{"observabilityStrategy": "none"}}' && sleep 120
+  changed_when: false
+
+
+- name: "Delete Grafana object"
+  ansible.builtin.shell:
+    cmd: |
+      oc delete grafana default && sleep 60
+  changed_when: false
+
+
+- name: "Delete Prometheus object"
+  ansible.builtin.shell:
+    cmd: |
+      oc delete prometheus default && sleep 60
+  changed_when: false
+
+
+- name: "Delete Alertmanager object"
+  ansible.builtin.shell:
+    cmd: |
+      oc delete alertmanager default && sleep 60
+  changed_when: false
+
+
+- name: "Delete ElasticSearch  object"
+  ansible.builtin.shell:
+    cmd: |
+      oc delete elasticsearches.elasticsearch.k8s.elastic.co elasticsearch && sleep 60
+  changed_when: false
+
+
+- name: "Verify that there no events SG pods"
+  ansible.builtin.shell:
+     cmd: |
+       oc get pods | grep event| wc -l
+  register: output
+  failed_when: "output.stdout|int !=0"
+  changed_when: false
+
+
+
+- name: "Verify that Grafana object didn't respawn"
```

Comment:
```suggestion
- name: "Verify that Grafana pod didn't respawn"
```

Reply:
Suggesting these description changes to help distinguish between the object that the operator acts on, and the pods that are a result of the operator's actions.